packages/api/src/controllers/stream.ts

```diff
@@ -1848,6 +1848,25 @@ app.get("/:id/info", authorizer({}), async (req, res) => {
   res.json(resp);
 });
 
+app.get("/:id/config", authorizer({ anyAdmin: true }), async (req, res) => {
+  let { id } = req.params;
+  let stream = await db.stream.getByPlaybackId(id);
+  if (!stream) {
+    stream = await db.stream.get(id);
+  }
+  if (!stream) {
+    res.status(404);
+    return res.json({
+      errors: ["not found"],
+    });
+  }
+
+  res.status(200);
+  res.json({
+    pull: stream.pull,
```

Comment:
Do we want to forward the whole pull? We could omit the `location` field if catalyst will make no use of it.
```suggestion
    pull: !stream.pull : null : {
      ...stream.pull,
      location: undefined
    }
```